assets/src/edit-story/components/library/panes/shapes/shapePreview.js

```diff
@@ -30,37 +30,51 @@ import { PAGE_WIDTH } from '../../../../constants';
 import createSolidFromString from '../../../../utils/createSolidFromString';
 import LibraryMoveable from '../shared/libraryMoveable';
 import { useUnits } from '../../../../units';
+import { themeHelpers } from '../../../../../design-system';
 
 // By default, the element should be 33% of the page.
 const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 3;
 const PREVIEW_SIZE = 36;
 
-const createGrid = ({ columns, gap, minWidth }) => css`
+const createGrid = ({ columns, columnGap = 12, rowGap = 24, minWidth = 50 }) => css`
   min-width: ${minWidth}px;
-  width: calc(${100 / columns}% - ${(gap * (columns - 1)) / columns}px);
+  width: calc(${100 / columns}% - ${(columnGap * (columns - 1)) / columns}px);
   margin-top: 0px;
-  margin-left: ${gap}px;
+  margin-left: ${columnGap}px;
   &:nth-of-type(n + ${columns + 1}) {
-    margin-top: ${gap}px;
+    margin-top: ${rowGap}px;
   }
   &:nth-of-type(${columns}n + 1) {
     margin-left: 0;
   }
 `;
 
 // Using button directly breaks the DOM nesting for tests.
-const Aspect = styled.div.attrs({ role: 'button' })`
+const Aspect = styled.button`
```

Comment:
I can see that this does not have specific hover styling [in Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235), but can you confirm with UX if this is actually the intention?

![Screen Shot 2021-03-16 at 11 47 27](https://user-images.githubusercontent.com/637548/111341120-9a0cca80-864f-11eb-8128-b9e54c63ea7a.png)


Seems like an oversight, as all our other buttons do have that.

Also, I think it should have `cursor:pointer` in here somewhere as well.